metadata/org.apache.curator/curator-client/5.5.0/reflect-config.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "condition":{"typeReachable":"org.apache.curator.retry.RetryForever"},
```

Comment:
This file doesn't contain any curator client metadata. It contains only `java.lang` related metadata which don't have to be added to this file.

Reply:
- Thanks for your reminder. Due to some unreasonable Warnings and failed nativeTests in the past 10 months, I always thought that these `java.**` and `sun.**` must be defined in GraalVM Reachability Metadata.  I've done further testing at https://github.com/linghengqian/graalvm-trace-metadata-smoketest/commit/ce5c7cb386e1a1905c3f3ebcba379b76703eb84b and https://github.com/apache/shardingsphere-elasticjob/pull/2268 , curator-client and curator-recipes of has none of failed nativeTest.  

- I will close this PR.  Other related PRs will be processed in the future. Maybe tomorrow.